<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="MarkSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>58</width>
     <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>7.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>212</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Line Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Aspect Radio:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Aspect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="LineSize">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="background">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_x">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>label_x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>label_y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_read">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>loadData</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rangeX">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rangeX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rangeY">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rangeY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Zero Line:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="zerolinex">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>58</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.989999999999782</double>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
   <property name="value">
    <double>1999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="zeroliney">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>58</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.989999999999782</double>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
   <property name="value">
    <double>350.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>6</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>6</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overlapC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>OverlapC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overlapL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>OverlapL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overlapR">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>OverlapR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_opt_marker">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OptMarker</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
